### PR TITLE
Adjustment for description of the list

### PR DESCRIPTION
**Pull request by benjaoming**
CC: @ericholscher trying to explain a bit more why this list is interesting :) if you like it, I think we should add it! It gives context to the topics below.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ A list of example projects, useful to learn from and for bootstrapping new docum
 <!-- description -->
 
 Read the Docs is a fully open-source platform that builds and publishes documentation. Read more
-on [readthedocs.org](https://readthedocs.org/) and [readthedocs.com](https://readthedocs.com/) (Read the Docs for Business).
+on [readthedocs.org](https://readthedocs.org/) and [readthedocs.com](https://readthedocs.com/) (Read the Docs for Business). Documentation projects have traditionally been part of a software project. However, many new documentation projects have emerged in science and academia. Therefore, we are compiling a list of inspirational uses of documentation technology, especially outside of the traditional field of software documentation. It's a work in progress, please feel invited to [contribute](#contributing)!
```
Comment (ericholscher) [inline README.md]: A reminder to not put everything in one line. It's hard to comment on a specific paragraph.

> Therefore, we are compiling a list of inspirational uses of documentation technology, especially outside of the traditional field of software documentation.

Is that the use of this page? I thought it was for all awesome docs projects we host?

Comment (benjaoming) [reply README.md]: It should be, I mean it still fits.

I'll try on two lines.. I get stressed by the "Awesome Linter", I'm sure it's gonna just say you can't have two lines in the description :)

Comment (benjaoming) [inline README.md]:
```suggestion
on [readthedocs.org](https://readthedocs.org/) and [readthedocs.com](https://readthedocs.com/) (Read the Docs for Business).

Documentation projects have traditionally been part of a software project. However, many new documentation projects have emerged in science and academia. Therefore, we are compiling a list of inspirational uses of documentation technology, especially outside of the traditional field of software documentation. It's a work in progress, please feel invited to [contribute](#contributing)!
```

Comment (benjaoming) [reply README.md]: Two lines worked, linter is appeased

> Is that the use of this page? I thought it was for all awesome docs projects we host?

Can you elaborate the comment? In my mind the use of "especially" means that we are highlighting a special aspect of why this list is motivated, but the general "awesome" aspect remains.

Comment (benjaoming) [reply README.md]: Split in several lines, linter ate it. Improvements welcome!

**Comment by ericholscher** [comment]
@benjaoming we should disable the linter, if it has arbitrary rules that go against our way of doing things. Also in general we don't merge things without a 👍, just as a note. This isn't a big change, but I don't think it's a good president to set.

**Comment by benjaoming** [comment]
@ericholscher the linter guarantees that we can adopt the list into the "grand" awesome lists, or maybe it doesn't but that was the idea.

If we scrap the linter and the general Awesome List format, we also get some new freedoms.

If we want consistent reviews on all PRs here, I need someone from the team to be dedicated to it, I already added 10 PRs with new entries.

**Comment by ericholscher** [comment]
> If we want consistent reviews on all PRs here, I need someone from the team to be dedicated to it, I already added 10 PRs with new entries.

I review everything that I get asked for review on. We have a process for asking for reviews, which will ensure that things get reviewed.

**Pull request merged**